conformance/conformance_suite.go

```diff
@@ -83,45 +83,62 @@ var _ = BeforeSuite(func() {
 func setupClients() error {
 	splitContexts := strings.Split(contexts, ",")
 	clients = make([]clusterClients, len(splitContexts))
-	for i, context := range splitContexts {
-		overrides := clientcmd.ConfigOverrides{ClusterDefaults: clientcmd.ClusterDefaults}
-		overrides.CurrentContext = context
+	accumulatedErrors := []error{}
 
-		clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &overrides)
+	for i, kubeContext := range splitContexts {
+		err := func() error {
+			overrides := clientcmd.ConfigOverrides{ClusterDefaults: clientcmd.ClusterDefaults}
+			overrides.CurrentContext = kubeContext
 
-		rawConfig, err := clientConfig.RawConfig()
-		if err != nil {
-			return err
-		}
+			clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &overrides)
 
-		name := context
-		if name == "" {
-			name = rawConfig.CurrentContext
-		}
+			rawConfig, err := clientConfig.RawConfig()
+			if err != nil {
+				return fmt.Errorf("error setting up a Kubernetes API client on context %s: %w", kubeContext, err)
```

Comment:
`errors.Wrap` is a bit simpler. Same below.

```suggestion
				return errors.Wrapf(err, "error setting up a Kubernetes API client on context %q", kubeContext)
```

Reply:
I know, but it’s not _that_ much simpler and we already have a different `errors` import…